Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -109,6 +113,52 @@ struct OdometerView: View {
 }
 
 #Preview {
-    OdometerView(userUID: "")
+    var viewModel = OdometerViewModel(userUID: nil)
+    let firstCar = createVehicle(id: "id1", name: "My 1st car")
+    viewModel.vehicles.append(firstCar)
```

Comment:
moved these to be added together with `viewModel.vehicles.append(contentsOf: [firstCar, secondCar])`  as opposed to individually